CMP1903M Assessment 1 Base Code/Input.cs

```diff
@@ -8,57 +8,72 @@ namespace CMP1903M_Assessment_1_Base_Code
 {
     public class Input
     {
+        private string? _input;
+        public string input { get { return (string)input; } }
         //Handles the text input for Assessment 1
 
         //Method: manualTextInput
         //Arguments: none
-        //Returns: string
+        //Returns: bool (if success or not)
         //Gets text input from the keyboard
-        public static void ManualTextInput(string Manual)
+        public bool ManualTextInput(string manual)
         {
+            Console.WriteLine("Is this correct: {0}Y/N: ", manual);
```

Comment:
I recommend using inline variables like so.
This can be applied to any `string` literal, whereas your current method may only be applied to `Console.WriteLine()`.
```suggestion
            Console.WriteLine($"Is this correct: {manual}Y/N: ");
```